packages/fury-adapter-oas3-parser/STATUS.md

```diff
@@ -72,7 +72,7 @@ Key:
 | responses | [~](#responses-object) |
 | callbacks | [✕](https://github.com/apiaryio/api-elements.js/issues/74) |
 | deprecated | ✕ |
-| security | [✕](https://github.com/apiaryio/api-elements.js/issues/77) |
+| security | ✓ |
```

Comment:
security isn't fully supported, should be ~ with link to nessecery objects below which show the unsupported components:

- provides an unsupported warning for openIdConnect type
- provides warning for unsupported bearerFormat property
- provides warning for unsupported openIdConnectUrl property